src/pages/MonthlyGood/LandingFormExperiment.vue

```diff
@@ -11,10 +11,10 @@
 				@custom-amount-updated="customAmountUpdated"
 				:split-pills="true"
 			/>
-			<span class="large-4 medium-4 small-12 columns select-category-label">
+			<span class="xlarge-5 columns select-category-label">
```

Comment:
Using the row/column classes isn't necessary for this component. You can remove the wrapping `<div class="row">` and remove the `xlarge-5`, `xlarge-7`, and `columns` classes from these spans.
Additionally, you can remove the
```
max-width: 8.25rem;
padding-right: 0;
```
from the css for `.select-category-label`.
Since these elements are spans, they will default to `display: inline` which will give you the appearance you're currently going for.